api/spec/integration/api/v2/platform/taxonomies_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'swagger_helper'
+
+describe 'Taxonomies API', swagger: true do
+  include_context 'Platform API v2'
+
+  resource_name = 'Taxonomy'
+  options = {
+    include_example: 'taxons,root',
+    filter_examples: [{ name: 'filter[name_eq]', example: 'Categories' }]
+  }
+
+  let(:id) { create(:taxonomy).id }
+  let(:records_list) { create_list(:taxonomy, 2) }
+  let(:valid_create_param_value) { build(:taxonomy).attributes }
+  let(:valid_update_param_value) do
+    {
+      name: 'Categories',
+      store_id: '2',
```

Comment:
```suggestion
```

@aleksandarpetrushev `store_id` will be ommited and replaced with the current store, it's not possible to overwrite store_id